builders/xcm/xc20/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: XC-20
```

Comment:
the title and description from the SEO weren't translated in the SEO PR so you could probably add it here:

```
---
title: Cross Chain Assets & XC-20s
description: Learn about cross chain assets, referred to as XC-20s, and how to interact with them through the Ethereum API on Moonbeam.
template: main.html
---
```